drivers/eks/templates.go

```diff
@@ -1,5 +1,6 @@
 package eks
 
+// This are the cloudformation templates used for eks clusters, when making edits here ensure that view whitespace is enabled
```

Comment:
`// These are the CloudFormation templates used for EKS clusters, when making edits here ensure the whitespace is correct.``